components/tabs/private/tab.jsx

```diff
@@ -76,6 +78,20 @@ class Tab extends React.Component {
 		 * If the Tabs should be scopped, defaults to false
 		 */
 		variant: PropTypes.oneOf(['default', 'scoped']),
+
+		/**
```

Comment:
Could you also add these props to `tabs/panel.jsx`, and then run `npm run build:docs` to generate the new documentation json file?  Thanks!